public/components/player/list/list.js

```diff
@@ -9,17 +39,53 @@ require("can/view/href/");
 
 var Player = require("bitballs/models/player");
 
-var ViewModel = exports.ViewModel = CanMap.extend({
+var ViewModel = exports.ViewModel = CanMap.extend(
+/** @prototype */
+{
 	define: {
+		/**
+		 * @property {Boolean} bitballs/components/player/list.isAdmin isAdmin
+		 * @parent bitballs/components/player/list.properties
+		 *
+		 * Configures whether or not admin specific features are enabled.
+		 **/
+		isAdmin: {
+			type: 'boolean',
```

Comment:
ah, well the type converter will help w/ what I wrote above.  But still should probably show people passing `true`.